pyaedt/modules/PostProcessor.py

```diff
@@ -601,7 +601,351 @@ def surfacePlotInstruction(self):
+
+        if families_dict is None:
+            families_dict = {"Freq": ["All"]}
+
+        solution_data = self.get_solution_data_per_variation(report_input_type, setup_sweep_name, ctxt, families_dict,
+                                                             expression)
+
+        if not solution_data:
+            print("No Data Available. Check inputs")
+            return False
+        return solution_data
+
+    @aedt_exception_handler
+    def create_rectangular_plot(self, expression="dB(S(1,1))", setup_sweep_name='', families_dict={"Freq": ["All"]},
+                                primary_sweep_variable="Freq", context=None, plotname=None, plottype=None):
+        """Create a 2D rectangular plot in AEDT.
+
+        Parameters
+        ----------
+        expression : str or list, optional
+            One or more formulas to add to the report. The default is value = ``"dB(S(1,1))"``.
+        setup_sweep_name : str, optional
+            Setup name with the sweep. The default is ``""``.
+        families_dict : dict, optional
+            Dictionary of all families including the primary sweep. The default is ``{"Freq": ["All"]}``.
+        primary_sweep_variable : str, optional
+            Name of the primary sweep. The default is ``"Freq"``.
+        context : str, optional
+            The default is ``None``.
+        plotname : str, optional
+            Name of the plot. The default is ``None``.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+        ctxt = []
+        if not setup_sweep_name:
+            setup_sweep_name = self._parent.nominal_sweep
+        if self.post_solution_type == "HFSS 3D Layout Design" or self.post_solution_type == "NexximLNA" or self.post_solution_type == "NexximTransient":
+            if "Freq" == primary_sweep_variable or "Freq" in list(families_dict.keys()):
+                did = 3
+            else:
+                did = 1
+            ctxt = ["NAME:Context", "SimValueContext:=",
+                    [did, 0, 2, 0, False, False, -1, 1, 0, 1, 1, "", 0, 0, "IDIID", False, "1"]]
+        elif context:
+            if type(context) is list:
+                ctxt = context
+            else:
+                ctxt = ["Context:=", context]
+
+        if type(expression) is not list:
+            expression = [expression]
+        if not setup_sweep_name:
+            setup_sweep_name = self._parent.nominal_sweep
+        if self.post_solution_type not in report_type:
+            print("Solution not supported")
+            return False
+        if not plottype:
+            modal_data = report_type[self.post_solution_type]
+        else:
+            modal_data = plottype
+        if not plotname:
+            plotname = generate_unique_name("Plot")
+        families_input = []
+        families_input.append(primary_sweep_variable + ":=")
+        if not primary_sweep_variable in list(families_dict.keys()):
```

Comment:
We can directly check if a key is in a `dict`.  For example:
```py
'a' in {'a': 1, 'b': 2}
```

```suggestion
        if not primary_sweep_variable in families_dict:
```